docs/core/dependency-loading/collect-details.md

```diff
@@ -30,7 +30,7 @@ The collected trace file can be viewed on Windows using the Events view in [Perf
 
 ## Example (on Windows)
 
-This example uses the [assembly loading extension points sample](https://github.com/dotnet/samples/tree/master/core/extensions/AssemblyLoading). The application attempts to load an assembly `MyLibrary` - an assembly that is not referenced by the application and thus requires handling in an assembly loading extension point to be successfully loaded.
+This example uses the [assembly loading extension points sample](https://github.com/dotnet/samples/tree/main/core/extensions/AssemblyLoading). The application attempts to load an assembly `MyLibrary` - an assembly that is not referenced by the application and thus requires handling in an assembly loading extension point to be successfully loaded.
```

Comment:
It didn't move to main yet, but I'm going to link to the samples browser instead as it's better anyway.

```suggestion
This example uses the [assembly loading extension points sample](https://docs.microsoft.com/samples/dotnet/samples/assembly-loading-extension-points/). The application attempts to load an assembly `MyLibrary` - an assembly that is not referenced by the application and thus requires handling in an assembly loading extension point to be successfully loaded.
```